src/style.css

```
@@ -0,0 +1,108 @@
+* {
+  margin: 0;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.card__image {
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 auto 4px;
+  font-weight: 500px;
```

```suggestion
  font-weight: 500;
```
через це і не застосовувалося))